configure.ac

```diff
@@ -468,6 +468,23 @@ AC_DEFUN([AC_PROG_CXX], [])
 AC_PROG_LN_S
 AC_PROG_INSTALL
 
+PRINT_VAR(JAVA_HOME)
+PRINT_VAR(JAVA)
+PRINT_VAR(JAVAC)
+PRINT_VAR(JAVAH)
+PRINT_VAR(JAR)
+PRINT_VAR(JNI_INCLUDES)
+
+AC_PATH_PROG(javahome,java, "/usr/bin")
+javahome=`dirname ${javahome}`
+javahome=`dirname ${javahome}`
```

Comment:
Why twice?
Do we  need this for anything later on?

Reply:
That was by accident, however I was slightly confused about the formatting